Tests/Marketplace/marketplace_services.py

```diff
@@ -750,6 +751,59 @@ def encrypt_pack(self, zip_pack_path, pack_name, encryption_key, extract_destina
         except subprocess.CalledProcessError as error:
             print(f"Error while trying to encrypt pack. {error}")
 
+    def decrypt_pack(self, encrypted_zip_pack_path, decryption_key):
+        """ decrypt the pack in order to see that the pack was encrypted in the first place.
+
+        Args:
+            encrypted_zip_pack_path (str): The path for the encrypted zip pack.
+            decryption_key (str): The key which we can decrypt the pack with.
+
+        Returns:
+            bool: whether the decryption succeeded.
+        """
+        try:
+            current_working_dir = os.getcwd()
+            extract_destination_path = f'{current_working_dir}/decrypt_pack_dir'
```

Comment:
It looks to me as if you might be accidentally adding the decrypted packs to a file that will later be stored somewhere else, such as the artifacts. What is the current working directory at this step?
Keeping this here in case we won't remove the pack anymore